spec/system/partner_system_spec.rb

```diff
@@ -256,6 +256,27 @@
           expect(rows.join).to have_text('7897', count: 2)
         end
       end
+
+      context "when viewing a partner's users" do
+        subject { url_prefix + "/partners/#{partner.id}" }
+        let(:partner) { create(:partner, name: "Partner") }
+        let(:partner_user) { partner.profile.users.first }
+        let(:invitation_sent_at) { partner_user.invitation_sent_at.strftime('%B %-d, %Y') }
+        let(:last_sign_in_at) { partner_user.last_sign_in_at.strftime('%B %-d, %Y') }
+
+        it 'can show users of a partner' do
+          visit subject
+
+          within("#partner-users") do
+            expect(page).to have_content(partner_user.name)
+            expect(page).to have_content(partner_user.email)
+            expect(page).to have_content(invitation_sent_at)
+            expect(page).to have_content(last_sign_in_at)
+            expect("Invitation Sent").to appear_before("Last Logged In")
+            expect(invitation_sent_at).to appear_before(last_sign_in_at)
```

Comment:
If this is the only place we used the `orderly` gem for, I actually suggest not including it at all. Orderly seems useful but I don't know if it would be widely used? Thoughts?

Reply:
I pushed for the inclusion of an explicit order check. Since we were using identical display styles for the two columns (and we list the headers separately from the values in the model), I could see myself easily changing the ordering of one (the header or the value) without remembering to change the order of the other, and, being the same display type/style, it would be very easy to miss the incorrect transposition when visually scanning the data.

If they are displayed in a way that would make a header ordering error much more obvious, then verifying that with a test would be less important.